components/package.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
A lot of this values are the ones of the parent repo, they should be changed to be in the components folder scope.